app/models/database.py

```diff
@@ -0,0 +1,57 @@
+from typing import Optional
+from boto3.dynamodb.conditions import Key
+from boto3.session import Session
+from pydantic import BaseModel
+
+
+#this file will contain connection to the database, and database logic
+
+
+TABLE_NAME = "LinkShortener"
+
+#connection parameters
+session = Session()
+dynamodb = session.resource("dynamodb", endpoint_url="http://localhost:8000") #local for now
```

Comment:
Just clarifying your localhost8000 is just a placeholder right? 

You should make a variable similar to "TABLE_NAME" for this magic string

Reply:
Yes, localhost is just for setup purposes,
Also, created the magic string, will upload shortly